src/seedu/addressbook/parser/Parser.java

```diff
@@ -1,28 +1,14 @@
 package seedu.addressbook.parser;
 
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import seedu.addressbook.commands.*;
```

Comment:
Imported classes should always be stated explicitly! If your IDE is automatically converting this, please do change the settings to prevent it from doing so.